devices.js

```diff
@@ -17524,6 +17524,28 @@ const devices = [
         extend: preset.light_onoff_brightness_colortemp({colorTempRange: [200, 370]}),
     },
 
+    // Matcall BV
+    {
+        zigbeeModel: ['ZG 401224'],
+        model: 'ZG 401224',
+        vendor: 'Matcall bv',
+        description: 'Matcall LED Dimmer driver',
+        exposes: [e.light_brightness()],
+        fromZigbee: [fz.on_off, fz.brightness, fz.ignore_basic_report],
+        toZigbee: [tz.light_onoff_brightness, tz.ignore_transition, tz.ignore_rate, tz.light_brightness_move,
+            tz.light_brightness_step, tz.level_config, tz.power_on_behavior],
+    },
+    {
+        zigbeeModel: ['ZG 430700', 'ZG  430700'],
+        model: 'ZG 430700',
```

Comment:
Please remove space for both model numbers (see bottom of page 26: https://www.matcall.nl/wp-content/uploads/documentatie/MC-Easy-systeem.pdf)

Reply:
These model numbers are the actual reported model numbers in the Zigbee stack. I guess it does make more sense to make them list as the documentation does. On their website they are also referred to without spaces. Will make changes!